scripts/test/Engineering/texture/TextureUtils/test_fitting_utils.py

```
@@ -276,606 +276,41 @@ def test_fit_initial_summed_spectra_ikeda_carpenter_fixes_params(
+            mock_instance = MagicMock()
+            mock_func_factory.Instance.return_value = mock_instance
+            mock_instance.createPeakFunction.return_value = MagicMock()
+
+            out_peak_func = mock_fit.return_value.Function.function.getFunction.return_value
+            out_peak_func.nParams.return_value = 3
+            out_peak_func.getParamName.side_effect = lambda i: ["I", "X0", "A"][i]
+            # getParameterValue is called with the "X0" name (for the centre) and with integer indices
+            # (when building the shared-params dict)
+            out_peak_func.getParameterValue.side_effect = lambda k: {"X0": 1.0, 0: 100.0, 1: 1.0, 2: 0.5}[k]
+
+            x0_lims, shared_params, all_wss = fit_initial_summed_spectra(
```

Ruff (0.16.1)

[warning] Unpacked variable `x0_lims` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)